app/models/true_work_income_validation.rb

```diff
@@ -0,0 +1,113 @@
+require 'truework'
+require 'base64'
+
+class TrueWorkIncomeValidation < IncomeValidation
+
+  def self.job_class
+  	TrueWorkIncomeValidationJob
+  end
+
+  def job_class
+    self.class.job_class
+  end
+
+  def initiate_validation
+    if !$truework_enabled
+      raise "Truework integration not enabled, cannot validate"
+    end
+
+  	update_attributes(status: :in_process)
+
+    begin
+      verification_request = Truework::VerificationRequest.create(
+        type: 'employment-income',
+        permissible_purpose: 'employee-eligibility',
+        target: {
+          first_name: input["first_name"],
+          last_name: input["last_name"],
+          social_security_number: input["social_security_number"],
+          contact_email: input["email"],
+          company: { id: input["company_id"] } # Applicant chooses company synchronously in UI
+        },
+        documents: [
+          # TODO: Will we support any document types?
+        ],
+        additional_information: "This is a demo/test conducted by U.S. Digital Response"
+      )
+
+      if verification_request[:state] == "canceled"
+        # This is an error.
+        output = {
+          error_info: verification_request[:cancellation_reason],
+          error_details: verification_request[:cancellation_details]
+        }
+        update_attributes(status: :error, output: output)
+      else
+        # Enqueued- we need to check back later to get the result
+        check_back_hours = 1
+        if verification_request[:turnaround_time]
+          check_back_hours = verification_request[:turnaround_time][:lower_bound]
+        end
+
+        verification_info = {verification_id: verification_request[:id]}

```

Comment:
Maybe `verification_request[:id]` should be stored in the verifications table, instead of (or in addition to) in the job in Redis.  That might make it much easier to manually restart the verification process if there's a problem or bug.

Reply:
I did this; makes it easier to restart jobs manually, etc.

Here's an example:
```
papua_development=# select * from validations;
                  id                  |             applicant_id             |           type           |                                                                         input                                                                          |                                     output                                      | status | decision |         created_at         |         updated_at
--------------------------------------+--------------------------------------+--------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------+--------+----------+----------------------------+----------------------------
 2e9bf13a-a7df-4a0e-9d34-cdbbc23448b9 | 01110aa3-73af-4ddb-beef-68a24e637c45 | TrueWorkIncomeValidation | {"email": "smedberg.usdr@gmail.com", "last_name": "Smedberg", "company_id": "20733", "first_name": "Michael", "social_security_number": "000-00-0000"} | {"verification_id": "AAAAAAADGb8AAa6qfU3YZCcTIkfYtMZe0wMA7EufZxAv0YNi2e6fgWoU"} |      1 |          | 2020-09-16 17:51:52.194234 | 2020-09-16 17:51:57.098879
```